app/CreateDeal.tsx

```diff
@@ -60,6 +63,21 @@ export function CreateDeal(props: {
                 />
               </div>
             </FormGroup>
+            <FormGroup>
+              <FormLabel>Unit</FormLabel>
+              {units.map((unit, index) => {
+                return (
+                  <FormCheckRadio
+                    key={index}
```

Comment:
I think if you look at the console, there'll might be some errors indicating that using index as the key is an antipattern (see https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key:~:text=throughout%20its%20lifetime.-,Pitfall,-You%20might%20be)
Use something else unique instead

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/30/commits/ee157cff020e9662af67a282076ce6f587886d4e